src/backend/dx12/src/lib.rs

```diff
@@ -781,6 +805,79 @@ impl hal::Instance for Instance {
                 }
             };
 
+            let mut format_properties = [f::Properties::default(); f::NUM_FORMATS];
+            for (i, props) in &mut format_properties.iter_mut().enumerate().skip(1) {
+                let format: f::Format = unsafe { mem::transmute(i as u32) };
+                let mut data = d3d12::D3D12_FEATURE_DATA_FORMAT_SUPPORT {
+                    Format: match conv::map_format(format) {
+                        Some(format) => format,
+                        None => continue,
+                    },
+                    Support1: unsafe { mem::zeroed() },
+                    Support2: unsafe { mem::zeroed() },
+                };
+                assert_eq!(winerror::S_OK, unsafe {
+                    device.CheckFeatureSupport(
+                        d3d12::D3D12_FEATURE_FORMAT_SUPPORT,
+                        &mut data as *mut _ as *mut _,
+                        mem::size_of::<d3d12::D3D12_FEATURE_DATA_FORMAT_SUPPORT>() as _,
+                    )
+                });
+                let can_buffer = 0 != data.Support1 & d3d12::D3D12_FORMAT_SUPPORT1_BUFFER;
+                let can_image = 0 != data.Support1 & (
+                    d3d12::D3D12_FORMAT_SUPPORT1_TEXTURE1D |
+                    d3d12::D3D12_FORMAT_SUPPORT1_TEXTURE2D |
+                    d3d12::D3D12_FORMAT_SUPPORT1_TEXTURE3D |
+                    d3d12::D3D12_FORMAT_SUPPORT1_TEXTURECUBE
+                );
+                if can_image {
+                    props.optimal_tiling |= f::ImageFeature::SAMPLED;
+                    // enable blit flags on non-compressed images only
```

Comment:
BC compressed must support blit src.
blit dst is implemented via rendertargets so should be set along with the color attachment flag. For later also for depth formats.